app/assets/stylesheets/govuk_publishing_components/components/_glance-metric.scss

```diff
@@ -0,0 +1,74 @@
+@import "govuk_publishing_components/individual_component_support";
+@import "govuk_publishing_components/components/mixins/media-down";
+
+.gem-c-glance-metric {
+  @include govuk-responsive-margin(6, "bottom");
+  border-top: 1px solid $govuk-border-colour;
+  border-bottom: 0;
+
+  &__heading {
+    @include govuk-responsive-margin(3, "top");
+    @include govuk-responsive-margin(1, "bottom");
+    @include govuk-responsive-padding(3, "top");
+    min-height: 2.6em;
+
+    @include media-down(tablet) {
+      min-height: 4em;
+    }
+
+    @include media-down(mobile) {
```

Comment:
I think `media-down` might be deprecated, as it seems to be related to Internet Explorer. Are you able to change these to use our more modern media queries, e.g.

```sass
  @include govuk-media-query($until: tablet) {
    // stuff here
  }
  @include govuk-media-query($from: tablet) {
    // stuff here
  }
```